recipes/hypercluster/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hypercluster" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "27c5ae4798e62ffa67d979c34fc98280368ec350c54ef1bd3402160e8a16bfd0"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - hdbscan >=0.8.24
+    - matplotlib >=3.1.0
+    - numpy >=1.16.4
+    - pandas >=0.24.2
+    - scikit-learn >=0.22.0
+    - scipy >=1.2.1
+    - seaborn >=0.9.0
+    - snakemake >=5.8.2
+
+test:
+  imports:
+    - hypercluster
+
+about:
+  home: "https://github.com/liliblu/hypercluster"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: ""
```

Comment:
Can you please add "LICENSE" here.

Reply:
Hi, yes i did it! it should be in the package.